pkg/cmd/roachtest/tests/rebalance_load.go

```diff
@@ -367,6 +369,14 @@ func makeStoreCPUFn(
 			}
 			// Take the latest CPU data point only.
 			cpu := result.Datapoints[len(result.Datapoints)-1].Value
+			// The datapoint is a float representing a percentage in [0,1.0]. Assert
+			// as much to avoid any surprises.
+			if cpu < 0 || cpu > 1 {
```

Comment:
Fixed this on latest push: should be > 1, as we multiply [0,1] to get [0,100] below.